webpack.common.babel.js

```diff
@@ -61,12 +66,21 @@ export default (env = {}) => ({
     plugins: [
         new Webpack.DefinePlugin({
             'process.env.NODE_ENV': env.dev ? `'development'` : `'production'` 
+        }),
+
+        new ExtractTextPlugin({
+            disable: !!env.dev,
+            filename: 'style.min.css'
         })
     ],
 
     output: {
         path: Path.resolve(__dirname, './dist'),
         publicPath: '/',
-        filename: 'app.min.js'
+        filename: '[name].bundle.js'
```

Comment:
Yeah, good call... I'm just used to doing it this way just in case there are multiple entries. However in this case simpler is probably better and I don't expect we'll have multiple entries here anytime soon.